web/content/js/nav.js

```diff
@@ -67,15 +58,57 @@ let navItems = [
 		href: "/Application_Centric_Deployment/index-foreman-el.html"
 	}]
 },{
-	title: "Katello documentation", items: [
+	title: "Foreman on Debian", items: [
+	{
+		title: "Release Notes",
+		href: "/Release_notes/index-foreman-deb.html"
+	},{
+		title: "Quickstart Guide",
+		href: "/Quickstart_Guide/index-foreman-deb.html"
+	},{
+		title: "Installation Guide",
+		href: "/Installing_Server_on_Red_Hat/index-foreman-deb.html"
+	},{
+		title: "Installing Smart Proxy",
+		href: "/Installing_Proxy_on_Red_Hat/index-foreman-deb.html"
+	},{
+		title: "Upgrading and Updating",
+		href: "/Upgrading_and_Updating/index-foreman-deb.html"
+	},{
+		title: "Deploying Foreman on AWS",
+		href: "/Deploying_on_AWS/index-foreman-deb.html"
+	},{
+		title: "Configuring Smart Proxies with a Load Balancer",
+		href: "/Configuring_Load_Balancer/index-foreman-deb.html"
+	},{
+		title: "Provisioning Guide",
+		href: "/Provisioning_Guide/index-foreman-deb.html"
```

Comment:
Not related to Debian, but in this guide I noticed it has `\http://smartproxy.example.com:8000/httpboot/ipxe-x64.efi` so something's wrong with the escaping.

Reply:
I will tag @maximiliankolb I am sure he is interested. Let's keep this one, I would like to maintain the Debian version too.

Reply:
I had a very quick look through it and nothing jumped out to me as wrong.

Reply:
Trying to fix this: https://github.com/theforeman/foreman-documentation/pull/971